src/simulations/spatial_parameters.jl

```diff
@@ -326,10 +326,10 @@ function default_spatially_varying_soil_parameters(
         regridder_kwargs = (; extrapolation_bc,),
     )
 
-    μ = FT(-6)
+    μ = FT(-5.11)
     K_sat .= masked_to_value.(K_sat, soil_params_mask, 10.0^μ)
 
-    ν .= masked_to_value.(ν, soil_params_mask, 1)
+    ν .= masked_to_value.(ν, soil_params_mask, 0.45)
```

Comment:
Can you add some short comments explaining where these numbers come from? (i.e. mean over the data)